Guide to visualize Wiz issues (#2290)
port-labs/port-docs: docs/guides/all/visualize-your-wiz-vulnerabilities.md

---
displayed_sidebar: null
description: Learn how to monitor Wiz issues and gain security insights using dashboards.
---

# Visualize your Wiz security issues

> This guide demonstrates how to set up a monitoring solution to gain visibility into security issues from your Wiz account. You’ll learn how to visualize vulnerabilities across your projects and track them over time using Port's **Wiz** integration.

Reviewer: ```suggestion
This guide demonstrates how to set up a monitoring solution to gain visibility into security issues from your Wiz account.  
We will see how to visualize vulnerabilities across your projects and track them over time using Port's **Wiz** integration.
```

<img src="/img/guides/wizVulnDashboard.png" border="1px" width="100%" />

## Common use cases

- Monitor open and resolved Wiz issues across projects.
- Understand the distribution of issues by severity and status.

## Prerequisites

This guide assumes the following:
- You have a Port account and have completed the [onboarding process](https://docs.port.io/getting-started/overview).

> - [Wiz ocean integration](https://docs.port.io/build-your-software-catalog/sync-data-to-catalog/code-quality-security/wiz) installed in your Port account.

Reviewer: ```suggestion
- Port's [Wiz integration](https://docs.port.io/build-your-software-catalog/sync-data-to-catalog/code-quality-security/wiz) is installed in your account.
```

## Visualize metrics

Once the Wiz data is synced to the catalog, we can create a dedicated dashboard in Port to monitor and analyze vulnerabilities using customizable widgets.

### Create a dashboard

1. Navigate to your [software catalog](https://app.getport.io/organization/catalog).
2. Click on the **`+ New`** button in the left sidebar.
3. Select **New dashboard**.
4. Name the dashboard **Wiz Security Insight**.
5. Select the `Wiz` icon.
6. Click `Create`.

We now have a blank dashboard where we can start adding widgets to visualize insights from the Wiz issues.

### Add widgets

> 

Reviewer: ```suggestion

In the new dashboard, create the following widgets:

```
